src/Context.php

```diff
@@ -237,25 +188,30 @@ public function notice($type, $message, \PhpParser\NodeAbstract $expr, $status =
      */
     public function syntaxError(\PhpParser\Error $exception, $filepath)
     {
-        $code = file($filepath);
+        return $this->addIssue('syntax-error', $exception->getMessage(), $exception->getStartLine(), $filepath);
+    }
+
+    /**
+     * @param $type
+     * @param $message
+     * @param $line
+     * @param string|null $filepath
+     * @return bool
+     */
+    protected function addIssue($type, $message, $line, $filepath = null)
```

Comment:
but anyway main question how pass categories? Lets add a parameter with default value to notice method, will be helpfull to pass normal category or categories of the Isue

And for errorLanguage lets pass `Issue::CATEGORY_BUG_RISK` to Issue